client/src/components/Description.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Description = props => (
+  <div id="description" style={props.border}><h1>Description</h1>
```

Comment:
I think you should lint this differently:
<div>
<h1>
</h1>
</div>

(with appropriate tabbing)